pkg/server/server_sql.go

```diff
@@ -919,6 +919,35 @@ func newSQLServer(ctx context.Context, cfg sqlServerArgs) (*SQLServer, error) {
 		)
 	}
 
+	// Start scheduled jobs daemon.
+	jobs.StartJobSchedulerDaemon(
+		ctx,
+		cfg.stopper,
+		cfg.registry,
+		&scheduledjobs.JobExecutionConfig{
+			Settings:         cfg.Settings,
+			InternalExecutor: cfg.circularInternalExecutor,
+			DB:               execCfg.DB,
+			TestingKnobs:     cfg.TestingKnobs.JobsTestingKnobs,
+			PlanHookMaker: func(opName string, txn *kv.Txn, user security.SQLUsername) (interface{}, func()) {
+				// This is a hack to get around a Go package dependency cycle. See comment
+				// in sql/jobs/registry.go on planHookMaker.
+				return sql.NewInternalPlanner(
+					opName,
+					txn,
+					user,
+					&sql.MemoryMetrics{},
+					execCfg,
+					sessiondatapb.SessionData{},
+				)
+			},
+			ShouldRunScheduler: func(ctx context.Context, ts hlc.ClockTimestamp) (bool, error) {
+				return cfg.isMeta1Leaseholder(ctx, ts)
+			},
```

Comment:
I suppose you could do `ShouldRunScheduler: cfg.isMeta1Leaseholder`.

There is a comment on the `isMeta1Leaseholder` field in the `sqlServerOptionalKVArgs` struct definition that we might want to update.

Reply:
Good point.  Comment updated.